_posts/2019-05-23-assessing-rails-upgrades.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Is this Rails Upgrade Project a good fit?"
+date: 2019-05-23 15:00:00
+categories: ["rails", "upgrades"]
+author: "emily"
+---
+
+Over the years we have seen a significant increase in the  demand for [Ruby on Rails](http://rubyonrails.org/) upgrades. That's why at [Ombu Labs](https://www.ombulabs.com) we decided to develop [FastRuby.io](https://fastruby.io), our Ruby on Rails upgrade service.
```

Comment:
I think this sounds a little better:

That's why a few years ago we decided to launch...